lib/init.coffee

```diff
@@ -55,12 +55,16 @@ module.exports =
       [javac argsfile](https://docs.oracle.com/javase/8/docs/technotes/tools/windows/javac.html#BHCCFGCD)
       that is located alongside with the .classpath file in the same directory.
       Contents of the argfile are passed to javac as arguments.'
+    debugMode:
```

Comment:
I know the dev mode - thanks. I did not consider asking users to change into the dev-mode.